# HG changeset patch
# User Rik <rik@octave.org>
# Date 1511830705 28800
#      Mon Nov 27 16:58:25 2017 -0800
# Branch stable
# Node ID 174a4426d441ef10d0a10df38950c38d6086538f
# Parent  3cdd2b6c78cb9ff171d3cc56d6059e1f284a79ff
stairs.m: Add example @code section showing what "indices of elements" means.

* stairs.m: Add example @code section showing what "indices of elements" means.

diff --git a/scripts/plot/draw/stairs.m b/scripts/plot/draw/stairs.m
--- a/scripts/plot/draw/stairs.m
+++ b/scripts/plot/draw/stairs.m
@@ -23,17 +23,18 @@
 ## @deftypefnx {} {} stairs (@dots{}, @var{prop}, @var{val}, @dots{})
 ## @deftypefnx {} {} stairs (@var{hax}, @dots{})
 ## @deftypefnx {} {@var{h} =} stairs (@dots{})
 ## @deftypefnx {} {[@var{xstep}, @var{ystep}] =} stairs (@dots{})
 ## Produce a stairstep plot.
 ##
 ## The arguments @var{x} and @var{y} may be vectors or matrices.
 ## If only one argument is given, it is taken as a vector of Y values
-## and the X coordinates are taken to be the indices of the elements.
+## and the X coordinates are taken to be the indices of the elements
+## (@code{@var{x} = 1:numel (@var{y})}).
 ##
 ## The style to use for the plot can be defined with a line style @var{style}
 ## of the same format as the @code{plot} command.
 ##
 ## Multiple property/value pairs may be specified, but they must appear in
 ## pairs.
 ##
 ## If the first argument @var{hax} is an axis handle, then plot into this axis,
